src/pages/dialog/components/InputTextArea.module.css

```diff
@@ -11,4 +11,49 @@
 	.TextField {
 		width: 50%;
 	}
+	/* max-width: 60vw; */
+}
+.chat-input__record-status .stop-recording {
+  margin-left: 5px;
+}
+.chat-input__record-status .stop-recording svg {
+  position: relative;
+  top: -1px;
+  width: 14px;
+  height: 14px;
+}
+.chat-input__record-status-bubble {
+  display: inline-block;
+  position: relative;
+  height: 10px;
+  width: 10px;
+  background-color: #f56b6b;
+  border-radius: 30px;
+  margin-right: 8px;
+}
+.chat-input__record-status-bubble::before {
```

Comment:
I think these styles are not used.

Reply:
these styles Valera write for future i think, so i don't delete them. i know they useless right now))